tests/regressiontests/storages.py

```diff
@@ -128,8 +129,9 @@ def test_write_png(self):
         from django_thumborstorage import storages
         from django.conf import settings
         filename = 'foundations/gnu.png'
-        content = ContentFile(open('%s/gnu.png' % IMAGE_DIR))
-        thumbor_file = storages.ThumborStorageFile(filename, mode="w")
+        file = open('%s/gnu.png' % IMAGE_DIR, "rb").read()
```

Comment:
I'd keep the one-liner to fix the read() issue.
Btw `file` isn't a File but a string in your code.
```python
content = ContentFile(open('%s/gnu.png' % IMAGE_DIR, "rb").read())
```

Same line 197.